src/platform/silabs/efr32/BUILD.gn

```diff
@@ -66,12 +68,23 @@ static_library("efr32") {
     "${silabs_platform_dir}/SystemPlatformConfig.h",
     "../../FreeRTOS/SystemTimeSupport.cpp",
     "../../SingletonConfigurationManager.cpp",
-    "BLEManagerImpl.cpp",
+#    "BLEManagerImpl.cpp",
     "ConfigurationManagerImpl.cpp",
     "KeyValueStoreManagerImpl.cpp",
     "PlatformManagerImpl.cpp",
   ]
 
+  if (chip_enable_ble_rs911x) {
+    sources += [
+        "rs911x/BLEManagerImpl.cpp",
+        "rs911x/wfx_sl_ble_init.h",
+        "rs911x/wfx_sl_ble_init.c",
+        "rs911x/rsi_ble_config.h",
+      ]
+    } else {
+    sources += [ "BLEManagerImpl.cpp" ]
```

Comment:
Because in the RS9116 before initializing the BLE we should initialize the Driver, WiSeConnect and radio that's why there is implementation difference.